avida-ts/packages/avida-tx-utils/src/sdjwt_utils.ts

```diff
@@ -0,0 +1,29 @@
+import { SDJwtVcInstance } from "@sd-jwt/sd-jwt-vc";
+import { generateSalt, digest } from "@sd-jwt/crypto-nodejs";
+import { subtle, createPrivateKey, sign } from "node:crypto";
+
+export const EdDSA = {
+  alg: "EdDSA",
+
+  async getSigner(privateKeyPEM: string) {
+    const privateKey = createPrivateKey(privateKeyPEM);
+
+    return async (data: any) => {
+      const encoder = new TextEncoder();
+      const signature = await sign(null, encoder.encode(data), privateKey);
+      return Buffer.from(signature).toString("base64url");
+    };
+  },
+};
+
+export async function get_sdjwt(privatePEM: string): Promise<SDJwtVcInstance> {
```

Comment:
let's keep the same style, camelCase for functions.

Reply:
fixed in https://github.com/nymlab/AVIDA/pull/6/commits/7ce3c1886296d88d19edcfd650036b81c8dcdf75